core/src/edu/cornell/gdiac/main/GameController.java

```diff
@@ -120,6 +120,22 @@ public class GameController implements Screen, ContactListener {
    */
   private int countdown;
 
+
+  /**
+   * Counter for keep track of meter
+   */
+  private float metercounter;
+
+  /**
+   * Constant for when stationary
+   */
+  private float stationaryconst = 0.25f;
+
+  /**
+   * Constant for freeze mechanic
+   */
+  private float freezeconst = 100;
```

Comment:
you don't need `const` at the end of a variable name, instead of `freezeconst` the variable name should describe what the variable actually means. For example, `FREEZE_SUSPICION_THRESHOLD` could be a better name here, since it tells me much more about what the variable actually does.